hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/scm/TestSCMInstallSnapshot.java

```diff
@@ -144,26 +148,24 @@ public void testInstallCheckPoint() throws Exception {
         scm.getPipelineManager().getPipelines().get(0).getId();
     scm.getScmMetadataStore().getPipelineTable().delete(pipelineID);
     scm.getContainerManager().deleteContainer(cid);
-    Assert.assertNull(
+    assertNull(
         scm.getScmMetadataStore().getPipelineTable().get(pipelineID));
-    Assert.assertFalse(scm.getContainerManager().containerExist(cid));
+    assertFalse(scm.getContainerManager().containerExist(cid));
 
     SCMStateMachine sm =
         scm.getScmHAManager().getRatisServer().getSCMStateMachine();
     sm.pause();
     sm.setInstallingSnapshotData(checkpoint, null);
     sm.reinitialize();
 
-    Assert.assertNotNull(
-        scm.getScmMetadataStore().getPipelineTable().get(pipelineID));
-    Assert.assertNotNull(
-        scm.getScmMetadataStore().getContainerTable().get(cid));
-    Assert.assertTrue(scm.getPipelineManager().containsPipeline(pipelineID));
-    Assert.assertTrue(scm.getContainerManager().containerExist(cid));
-    Assert.assertEquals(100, scm.getScmMetadataStore().
+    assertNotNull(scm.getScmMetadataStore().getPipelineTable().get(pipelineID));
+    assertNotNull(scm.getScmMetadataStore().getContainerTable().get(cid));
+    assertTrue(scm.getPipelineManager().containsPipeline(pipelineID));
+    assertTrue(scm.getContainerManager().containerExist(cid));
+    assertEquals(100, scm.getScmMetadataStore().
         getTransactionInfoTable().get(OzoneConsts.TRANSACTION_INFO_KEY)
         .getTransactionIndex());
-    Assert.assertEquals(100,
+    assertEquals(100,
```

Comment:
Nit: Now that the line length is increased to 120, We can use that here and in other changes as well ;)

Reply:
This is a good news to see the allowed line length increased to 120. This will improve readability. Thanks.